Doc/whatsnew/3.7.rst

```diff
@@ -604,3 +604,21 @@ CPython bytecode changes
 
 * Added two new opcodes: :opcode:`LOAD_METHOD` and :opcode:`CALL_METHOD`.
   (Contributed by Yury Selivanov and INADA Naoki in :issue:`26110`.)
+
+
+Documentation
+=============
+
+.. _whatsnew37-pep545:
```

Comment:
Why do you add this link?

Reply:
I spotted every titles about PEPs had a link so I just added a link too.

Reply:
I see. Just reformat to separate the two links of a language.

Reply:
Oops, my comment was for the http links below.

Reply:
done